lib/external_data_sets_two_electric_boogaloo.rb

```diff
@@ -0,0 +1,49 @@
+class FacilityObjects
+  def initialize
+    @facilities_list = []
+  end
+  
+  def create_facilities(api_data)
+    api_data.each do |facility_data|
+      fac_data = dmv_state_check(facility_data)
+      facility = Facility.new(fac_data)
+      @facilities_list << facility
+    end
+    @facilities_list
+  end
+
+  def dmv_state_check(api_data)
+    if api_data[:state] == "CO"
+      dmv_co_state_nomenclature(api_data)
+    elsif api_data[:state] == "NY"
+      dmv_ny_state_nomenclature(api_data)
+    else
+      dmv_mo_state_nomenclature(api_data)
+    end
+  end
```

Comment:
Nice! I really like this helper method that you created. I also like that you're filtering which method to call based on which state is included in the api data.